graph.py

```diff
@@ -0,0 +1,92 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import math
+import random
+
+#Input
+
+print("Please choose type of function\n"
+      "1 - Linear\n"
+      "2 - Sinus\n"
+      "3 - Log\n"
+      "4 - Random\n")
+a = [0,0]
+c = 0
+d = 0
+s = int(input())
+
+if s == 1 or s == 2 or s == 3:
+  print("Please, enter x array")
+  a = list(map(int, input().split()))
+  if s == 1:
+      print("Please, enter additional arguments k and b for linear function")
+      c= int(input())
+      d= int(input())
+
+
+
+def linFunction(x,k,b):
```

Comment:
I think you should extract support functions to the separate file/module since it's a good practice to follow single responsibility principle